src/Command/InfectionCommand.php

```diff
@@ -52,6 +52,7 @@
 final class InfectionCommand extends BaseCommand
 {
     const CI_FLAG_ERROR = 'The minimum required %s percentage should be %s%%, but actual is %s%%. Improve your tests!';
+    const COMMAND_NAME = 'run';
```

Comment:
If we are gonna do this here, should we do it for all our commands?

Reply:
There's `$command->getName()`

Reply:
because I don't want to compare it with a string?